bsip-0058.md

```diff
@@ -67,17 +67,221 @@ related poll worker proposals, including FOR and AGAINST, will be created for vo
+|-|
+|New supply cannot be created for a globally-settled asset.|
+
+#### Updating Collateral behind a Debt Position
+
+It is still possible to update the collateral behind a debt position.
+
+|Comparison to Globally Settled Asset|
+|-|
+|Debt positions cannot be updated for a globally-settled asset because they are closed when global settlement is triggered.|
+
+
+#### Destroying/Closing a Debt Positions
+
+It is still possible to close (pay back) a debt position.
+
+|Comparison to Globally Settled Asset|
+|-|
+|Debt positions are automatically closed when global settlement is triggered.|
+
+
+#### <div id="gsp-activated-margin-calls" /> Margin Calls
+
+Debt positions that were margin called prior to the activation of GSP, will remain margin called until their margin call position is matched or until their collateral ratio is raised by either adding collateral or by the price of the collateral increasing.
+
+_New debt positions (i.e. debt positions created after activation of GSP) are less likely to be margin called during GSP than without GSP because the feed price is protecting the least-collateralized debt position's collateral ratio from falling._
+
+|Comparison to Globally Settled Asset|
+|-|
+|Margin calls are not possible because debt positions are closed when global settlement is triggered.|
+
+
+#### <div id="lower-risk-for-new-debt" />New Borrowers Have a Lower Risk
+
+During GSP, a decrease in the feed price is limited by the need to protect _the least-collateralized debt position's CR from falling below MSSR_.  The feed price can decrease only when the least collateralized debt position's CR increases (i.e. which will cause its debt-denominated call price to decrease).  This CR increase can occur either by the debt position's owner (i.e. the borrower) increasing the collateral, or by its margin call order getting partly filled; partial filling of a margin call increases its CR as perceived by the blockchain.
+
+In addition, during significant downtrends of collateral pricing, many debt positions are often being margin called which create large "walls" on the DEX order book which must be filled before P<sub>gs</sub> can decrease significantly.
+
+Therefore, while GSP is activated is a relatively low-risk time for new debt positions to be created because of the difficulty in triggering margin calls on them.
+
+##### <div id="mitigation-attempt-against-new-supply" /> Mitigation Attempt: MCR-Adjustment
+
+This increased supply might be considered undesirable by some because this new supply might decrease the [reference-denominated price](#gsp-activated-reference-denominated-price) if it is offered on external markets.
+
+This opportunity can be mitigated by increasing the MCR to make the borrowing of new supply to be ever more expensive.  But increasing the MCR also has the additional effect of increasing the call price of the least collateralized debt position which then forces feed publishers to increase the feed price.
```

Comment:
" then forces feed publishers to increase the feed price" is not correct.

Reply:
This error is now corrected